tests/oserror_aliases_test.py

```diff
@@ -8,6 +8,72 @@
 from pyupgrade import FindPy3Plus
 
 
+@pytest.mark.parametrize('alias', FindPy3Plus.OS_ERROR_ALIASES)
+@pytest.mark.parametrize(
+    ('tpl', 'expected'),
+    (
+        (
+            'try:\n'
+            '    pass\n'
+            'except {alias}:\n'
+            '    pass\n',
+
+            'try:\n'
+            '    pass\n'
+            'except OSError:\n'
+            '    pass\n',
+        ),
+        (
+            'try:\n'
+            '    pass\n'
+            'except ({alias},):\n'
+            '    pass\n',
+
+            'try:\n'
+            '    pass\n'
+            'except OSError:\n'
+            '    pass\n',
+        ),
+        (
+            'try:\n'
+            '    pass\n'
+            'except ({alias}, KeyError, OSError):\n'
+            '    pass\n',
+
+            'try:\n'
+            '    pass\n'
+            'except (OSError, KeyError):\n'
+            '    pass\n',
+        ),
+        (
+            'try:\n'
+            '    pass\n'
+            'except ({alias}, OSError, IOError):\n'
+            '    pass\n',
+
+            'try:\n'
+            '    pass\n'
+            'except OSError:\n'
+            '    pass\n',
+        ),
+        (
+            'try:\n'
+            '    pass\n'
+            'except({alias}, OSError, IOError):\n'
+            '    pass\n',
+
+            'try:\n'
+            '    pass\n'
+            'except OSError:\n'
+            '    pass\n',
+        ),
+    ),
+)
+def test_fix_oserror_aliases_try(alias, tpl, expected):
```

Comment:
can you add a test where the tuple spans multiple lines?

Reply:
sure